Dependencies/napi/napi-jsi/CMakeLists.txt

```diff
@@ -16,4 +16,8 @@ target_include_directories(napi
 target_link_to_dependencies(napi
     PUBLIC jsi)
 
+if(WIN32)
+    target_compile_definitions(napi PRIVATE _SILENCE_CXX17_CODECVT_HEADER_DEPRECATION_WARNING)
```

Comment:
Probably should make this PUBLIC instead of PRIVATE and then you don't need to add it to all the remaining cmakelists.

Reply:
ah that explains the headache :P